test/NewsService.Business.UnitTests/NewsService.Business.UnitTests.csproj

```diff
@@ -2,6 +2,10 @@
 
   <Import Project="$(ProjectDir)..\..\Common.Configurations.targets" />
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="5.0.5" />
+  </ItemGroup>
+
```

Comment:
Remove

Reply:
i use this package in  edit command tests